### PR TITLE
Create label class and implement method

### PR DESCRIPTION
**Pull request by biruk-bereka**
## In this PR I:
**Create a Label class with an association to the Item class (in a separate .rb file).**
- All Label class properties visible in the diagram defined and set up in the constructor method.
- Implemented method:
    - add_item method in the Label class
        - take an instance of the Item class as an input
        - add the input item to the collection of items
        - add self as a property of the item object (by using the correct setter from the item object)

### CODE REVIEW

File: classes/label.rb [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
     @color = color
-    @items = []
+    @items = items
   end
 
   def add_item(item)
-    items.push(item)
-    item.label = self
+    @items.push(item)
+    @item.label = self
```
Comment (kensteph) [inline classes/label.rb]: Instead of  @item.label = self put item.label = self

**Review by kensteph — CHANGES_REQUESTED** [review]
Hi @biruk-bereka  

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some improvements, but you are almost there!

almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: classes/label.rb [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
+  attr_reader :items, :id
 
-  def initialize(_id, title, color, _items)
+  def initialize(title, color, items)
     @id = Random.rand(1..100)
     @title = title
     @color = color
-    @items = []
+    @items = items
   end
```
Comment (kensteph) [inline classes/label.rb]: Here is the best way to set up the association between Label and Item classes.

`def initialize(title, color)
    @id = Random.rand(1..100)
    @title = title
    @color = color
    @items = []
  end
`

**Review by kensteph — CHANGES_REQUESTED** [review]

File: classes/label.rb (hunk 2 above)
Comment (biruk-bereka) [reply classes/label.rb]: Thank you @kensteph for your review!

**Review by kensteph — APPROVED** [review]
Hi @biruk-bereka , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good  association implementation

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**